src/main/java/com/icthh/xm/uaa/config/LepAppEventListenerConfiguration.java

```diff
@@ -21,16 +23,29 @@
 
     @Bean
     XmUaaLepProcessingApplicationListener buildLepProcessingApplicationListener(
-                    AccountService accountService,
-                    TenantConfigService tenantConfigService,
-                    UserLoginService userLoginService,
-                    ProfileEventProducer profileEventProducer,
-                    MailService mailService,
-                    @Qualifier("loadBalancedRestTemplate") RestTemplate restTemplate,
-                    UserService userService,
-                    CommonsService commonsService) {
+        AccountService accountService,
```

Comment:
what reason to change count of space?

Reply:
Code formatting for the growing number of arguments, please see the difference between:
methodName(name,name2,name3)
and
methodName(
name,
name1,
name3
)
If you have any requirements please make sure that they are declared in the check style code validation task, thanks!